tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -104,18 +104,21 @@ module.exports = {
      *
      * @param {string} fileName
      */
-    deleteFile: async function (fileName) {
-      await this
+    deleteFile: function (fileName) {
+      return this
         .waitForFileVisible(fileName)
         .useXpath()
         .performFileAction(fileName, 'delete')
         .waitForElementEnabled(
           this.elements.deleteFileConfirmationBtn.selector,
           'css selector'
         )
-
+        .clickDeleteButton()
```

Comment:
.useCss( )

Reply:
@jasson99 ```clickDeleteButton()``` already returns with ```useCss()```, so attaching ```useCss()``` here makes no sense.

Reply:
does that work in other cases where we need to use deleteFile( ) for other test cases

Reply:
okay I understood now